gpuHuffmanConstruction.h

```diff
@@ -15,6 +15,8 @@
 #include <string>
 #include <vector>
 #include <functional>
+#include <numeric>
+#include <bitset>
 
```

Comment:
I am wondering why you added these two headers. Does missing these headers lead to compiler errors?